### PR TITLE
Rub 14 Add login component

### PR DESCRIPTION
**Pull request by StepanKovalyshyn**
# REVIEWERS
- [ ] @UAVova 
- [ ] @VR333  

# DESCRIPTION
Login component added.

# TEST COVERAGE
no tests

# EXPERT REVIEW
- [ ] @rokor

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,24 @@
...
+}
+
+html {
+    height: 100%
+}
+
+body {
+    height: 100%;
+    margin: 0;
+    background: url("1.jpg");
```
Comment (okohuch) [inline app/assets/stylesheets/application.css:30]: tab should be 2 spaces

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
-# ApplicationHelper
 module ApplicationHelper
+	def header(text)
+  	content_for(:header) { text.to_s }
```
Comment (okohuch) [inline app/helpers/application_helper.rb:3]: fix alignment of the method

File: app/assets/javascripts/components/loginForm.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+              <fieldset>
+                <legend>Log in</legend>
+                <div className="box">
+                    <input className="email"
+                           id="user_email"
+                           type="email"
+                           value={this.state.email}
+                           onChange={e => this.setState({email: e.target.value})}
+                           name="user[email]"
+                           placeholder="Enter email"
```
Comment (okohuch) [inline app/assets/javascripts/components/loginForm.jsx:46]: Do you have I18n in the application?

**Comment by rokor** [comment]
DEV branch  inside your PR would not event be reviewed!

**Pull request closed without merging**